eZ/Publish/Core/Persistence/Cache/URLHandler.php

```diff
@@ -110,15 +117,8 @@ public function loadByUrl($url)
      */
     public function findUsages($id)
     {
-        $cache = $this->cache->getItem('url', $id, 'usages');
-
-        $usages = $cache->get();
-        if ($cache->isMiss()) {
-            $this->logger->logCall(__METHOD__, ['url' => $id]);
-            $usages = $this->persistenceHandler->urlHandler()->findUsages($id);
-            $cache->set($usages)->save();
-        }
+        $this->logger->logCall(__METHOD__, ['url' => $id]);
 
-        return $usages;
```

Comment:
TODO NOTE for later: It should be rather a easy to cache this in 7.0, as you can tag it with content-id and url-id stuff making sure it is cleared on both url cand content changes.


However in 6,x you should probably open PR to remove it, as cache won't be cleared if content is deleted or updated for instance.